internal/backend/temporalclient/client.go

```diff
@@ -95,7 +96,8 @@ func (c *impl) startDevServer(ctx context.Context, cfg *Config, opts client.Opti
 	return nil
 }
 
-func (c *impl) Temporal() client.Client { return c.client }
+func (c *impl) Temporal() client.Client               { return c.client }
+func (c *impl) ScheduleClient() client.ScheduleClient { return c.client.ScheduleClient() }
```

Comment:
why is this needed? just use Temporal().ScheduleClient()

Reply:
done